src/app/design/styled-button/styled-button.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { StyledButtonComponent } from './styled-button.component';
+
+describe('StyledButtonComponent', () => {
```

Comment:
With version 3, we want to encourage the use of unit tests. Some basic tests for the flavors and the "disabled"-property would be nice and won't take much time! :)

Reply:
Everything else is fine